Packs/CommonScripts/Scripts/SetGridField/SetGridField.yml

```diff
@@ -30,6 +30,10 @@ args:
   predefined:
   - 'false'
   - 'true'
+- defaultValue: '*'
+  description: 'Keys to retrieve from nested dictionaries. Keys will not be columns correlated. Default is all keys. **Note**: when the number of values exceeds the number of columns, it truncates the last values that are outside the range for table.'
```

Comment:
let's update the README.md file with a detailed example.
I also think we should elaborate on the difference between this argument and `unpack_nested_elements` arg, as they sound very similar:
- maybe rename this one to `nested_keys_to_display`?
- what happens when we use both arguments?

Reply:
We cant use both, added clarification to the readme